CHANGELOG.md

```diff
@@ -2,12 +2,13 @@
 ### New Features
 
 ### Bugfixes
+- <a href="https://github.com/openscope/openscope/issues/1617" target="_blank">#1617</a> - Add LXJ to fix crash at KIAD
 
 ### Enhancements & Refactors
 
 # 6.18.0 (July 1, 2020)
 ### New Features
-- <a href="https://github.com/openscope/openscope/issues/1236" target="_blank">#1523</a> - Re-introduce Washington Dulles International Airport (KIAD); add Mountain Air Cargo (MTN) and CommutAir (UCA)
+- <a href="https://github.com/openscope/openscope/issues/1523" target="_blank">#1523</a> - Re-introduce Washington Dulles International Airport (KIAD); add Mountain Air Cargo (MTN) and CommutAir (UCA)
```

Comment:
You're killin me, @j-651...

Reply:
I linked it to the issue as that was how the one below was done 😂

Reply:
Oh god then the link was right and the text was wrong, and I changed the wrong one. Went from one right one wrong to both wrong.

Note that the issue number is BOTH in the link AND in the text that is shown to the user in the same line-- so you have to replace both.

I can fix it at the next pull request.

Reply:
Yeah I don’t know why I only changed the link and left the text as it was when I copied it. That’s what happens when I try to do the changelog ahead of time!

Both should be #1236.